plugin-server/src/worker/plugins/loadPluginsFromDB.ts

```diff
@@ -25,6 +25,17 @@ const loadPluginsTotalMsSummary = new Summary({
     percentiles: [0.5, 0.9, 0.95, 0.99],
 })
 
+// These are plugins that do I/O (fetch, cache, storage) that we want to deprecate. For now, we
+// want to at least ensure they don't run for "personless" ($process_person=false) events.
+const personlessPluginSkipList = [
+    'https://github.com/posthog/currency-normalization-plugin',
+    'https://github.com/posthog/event-sequence-timer-plugin',
+    'https://github.com/posthog/first-event-today',
+    'https://github.com/posthog/first-time-event-tracker',
+    'https://github.com/posthog/flatten-properties-plugin',
+    'https://github.com/posthog/mailboxlayer-plugin',
+]
```

Comment:
I used URLs because they are the best globally unique ID for plugins across regions, and this list should hopefully go away soon enough...